index.html

```diff
@@ -7,7 +7,7 @@
 <body>
 
   <p>Breakfast</p>
-  <table class="meals" id="breakfast-foods">
+  <table class="meals" id="Breakfast">
```

Comment:
are ids normally lowercase? and I see you changed these to uppercase, but i didn't necessarily see in the code where/why it had to be uppercase? no criticism at all, just wondering.